libs/domain/cart/src/mixins/cart.mixin.ts

```diff
@@ -48,6 +54,15 @@ export const CartComponentMixin = <
 
     @asyncState()
     protected totalQuantity = valueType(this.cartController.getTotalQuantity());
+
+    protected $isEmpty = signal(this.cartController.isEmpty(), false);
```

Comment:
Is it convention start signals name with `$`?

Reply:
generally there is no such a convention for signals as I'm aware, and for now it was only like this to make it possible to add them to mixins alongside asyncValue versions (which we will remove in the future, if all goes well), but we will discuss it, as it has some benefits